### PR TITLE
[Needs PO approval] Add dropdown to set a status for an errored workflow item (design needed - blocker)

### PR DESCRIPTION
**Pull request by jcoyne**
***DO NOT MERGE*** without approval from Product Owner. 

Ref #1346

Ref #1516

### CODE REVIEW

**Comment by ndushay** [comment]
This looks good but I think it needs to be deployed to stage and taken for a spin by @andrewjbtw or someone before it gets merged to master.  Also, before and after pix would be great here ...

(I'm psyched to see this functionality get added, btw!)

File: app/presenters/workflow_process_presenter.rb [hunk 1]
```diff
@@ -23,25 +17,40 @@ def elapsed
...
-    form_tag item_workflow_path(pid, workflow_name), method: 'put' do
-      hidden_field_tag('process', name) +
-        hidden_field_tag('status', new_status) +
-        button_tag('Set to ' + new_status, type: 'submit')
+  def status_action
+    case status
+    when 'error'
+      error_choices
+    when 'waiting'
+      completed_button('completed')
```
Comment (ndushay) [inline app/presenters/workflow_process_presenter.rb]: could this also be 'skipped'?  I'm not clear on the difference ...

Comment (jcoyne) [reply app/presenters/workflow_process_presenter.rb]: I'm not sure I understand.  This is the existing functionality for "waiting".  In https://github.com/sul-dlss/argo/issues/1346#issuecomment-521484228 we agreed the ticket was only affecting rows with the "error" status.

**Comment by ndushay** [comment]
Houston, I think we have a problem:  argo-stage shows this:

image

what is the "assign" button??

Also -- I suspect this will also need to be in a feature branch.

**Comment by jcoyne** [comment]
Assign is the submit button. As in assign this status

**Comment by ndushay** [comment]
Up to @andrewjbtw to sign off on UI, then.  Thanks for explanation.

**Comment by andrewjbtw** [comment]
The functionality looks good, but I would like to revisit the UI for this. We definitely want a mechanism to confirm the choice, but I was initially confused by the "Assign" label.

Also, depending on the layout of the workflow view, the button is sometimes on the right of the dropdown and sometimes below it. This seems to be related to how word wrapping applies to the process names on the left. Example from druid:hx447jk4615

Screen Shot 2019-08-16 at 9 22 37 AM

**Comment by ndushay** [comment]
I would like one more set of eyes on this.

**Pull request merged**